src/pipelines/vaccinations/ch_authority.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Dict
+from pandas import DataFrame
+from lib.data_source import DataSource
+from lib.utils import table_rename
+
+
+class SwitzerlandDataSource(DataSource):
+    def parse_dataframes(
+        self, dataframes: Dict[Any, DataFrame], aux: Dict[str, DataFrame], **parse_opts
+    ) -> DataFrame:
+
+        # Rename the appropriate columns
+        data = table_rename(
+            dataframes[0],
+            {
+                "date": "date",
```

Comment:
IFUC this will only be one row. I'm not sure that we can consume latest-date only data in the same way as historical data sources, I believe the vaccination data is intended to be time-series. @owahltinez will this auto-cache the previous data in some way?

Reply:
turns out https://www.covid19.admin.ch/api/data/context contains key ['sources']['individual']['csv']['vaccDosesAdministered']. the value for that key (which is https://www.covid19.admin.ch/api/data/20210216-qqpgbdwv/sources/COVID19VaccDosesAdministered.json today) is a file url which contains historical vaccine data